Headers/state_grid_mod.F90

```diff
@@ -37,8 +37,9 @@ MODULE State_Grid_Mod
      !----------------------------------------
      ! User-defined grid fields
      !----------------------------------------
-#if defined( MODEL_WRF )
+#if defined( MODEL_WRF ) || defined( MODEL_CESM )
```

Comment:
Could you add a clarifying comment in this block with more about what these are? Does each CPU have a different ID while root ID is the same for all? Or does each domain have a fixed ID and is handled by multiple CPUs such that multiple CPUs have the same ID?